hack/arktos-worker-up.sh

```diff
@@ -47,8 +56,35 @@ if [[ ! -s "${KUBELET_CLIENTCA}" ]]; then
     die "arktos worker node failed to start."
 fi
 
+if [[ ! -s "${KUBE_PROXY_KUBECONFIG}" ]]; then
```

Comment:
this block seems not right.

Reply:
Line60 needs rewording - kube-proxy kubeconfig file not found at ${KUBE_PROXY_KUBECONFIG}. Why this block is not right?

Reply:
discussed offline; the echo "gcloud ..." is fine as example here.